app/models/comment.rb

```diff
@@ -1,4 +1,12 @@
 class Comment < ApplicationRecord
+  after_save :update_comment_counter
+
   belongs_to :author, class_name: 'User', foreign_key: 'author_id'
```

Comment:
- [OPTIONAL] I wish we could realize that there is no need of defining the foreign key here. The User model has taken care of this already. Now, this becomes redundant. Having told rails that this model belongs to the `author`, the primary key will automatically be created.
```suggestion
  belongs_to :author, class_name: 'User'
```

Reply:
Done!